core/src/main/java/org/apache/calcite/DataContext.java

```diff
@@ -42,17 +44,17 @@
   /**
    * Returns a sub-schema with a given name, or null.
    */
-  SchemaPlus getRootSchema();
+  @Nullable SchemaPlus getRootSchema();
 
   /**
```

Comment:
Can we use `javax.annotation.Nullable` instead.

Reply:
`javax.annotation.Nullable` does not support cases like `List<javax.annotation.Nullable String>`